src/Interface/Filter/Filter.ts

```diff
@@ -0,0 +1,14 @@
+import Operator from '../../Type/Operator';
+
+export interface Filter extends Array<any> {
+  0: Operator;
+}
+
+export interface Negation extends Filter {
```

Comment:
Is it common in TS to have several ``interface`` definitions in one file?

Reply:
Im not sure about that. I just did it as it was so little code. But i see the advantages of splitting these into mutliple files. I ll change this.